packages/ebayui-core/src/components/ebay-video/examples/auto-play-viewport.marko

```diff
@@ -0,0 +1,51 @@
+class {}
+
+<style>
```

Comment:
I know this is only an example so it doesn't _really_ matter, but in Marko 5 you should **never** use a literal `<style>` tag. Always a `style {}` block instead.